android/app/build.gradle

```diff
@@ -31,6 +31,12 @@ if (keystorePropertiesFile.exists()) {
     keystoreProperties.load(new FileInputStream(keystorePropertiesFile))
 }
 
+
+//
+// Project won't compile for android if you don't have google-services.json and key.properties.
```

Comment:
Please specify the location required for each file.

Reply:
Good idea!